vercel.json

```diff
@@ -657,27 +653,11 @@
       "source": "/guides/deployment/deploying-to-netlify",
       "destination": "/docs/guides/deployment/deployment-guides/deploying-to-netlify"
     },
-
-    {
-      "source": "/guides/general-guides/database-workflows/cascading-deletes/:any*",
-      "destination": "/docs/guides/database/advanced-database-tasks/cascading-deletes/:any*"
-    },
-
     {
       "source": "/guides/general-guides/database-workflows/data-validation/:any*",
       "destination": "/docs/guides/database/advanced-database-tasks/data-validation/:any*"
     },
 
-    {
-      "source": "/guides/general-guides/database-workflows/sql-views-postgres",
-      "destination": "/docs/guides/database/advanced-database-tasks/sql-views-postgres"
-    },
-
-    {
-      "source": "/guides/general-guides/database-workflows/sql-views-mysql",
-      "destination": "/docs/guides/database/advanced-database-tasks/sql-views-mysql"
-    },
```

Comment:
Maybe redirect all the Views URLs to the GitHub issue instead?

Reply:
Added in these redirects